lib/services/template-service.ts

```diff
@@ -302,32 +220,32 @@ export class TemplateService implements ITemplateService {
                 }
             });
         } else {
-            console.error('Missing templates directory');
+            console.error("Missing templates directory");
         }
     }
 
-    public getPageTemplateDetails(templateName: string) {
-        let version: string,
-            flavor: string,
-            description: string,
-            templateDetails: any;
+    getPageTemplateDetails(templateName: string) {
+        const version: string = "";
+        const flavor: string = "";
+        const description: string = "";
+        let templateDetails: any;
 
-        return new Promise(function (resolve, reject) {
+        return new Promise((resolve, reject) => {
             try {
-                //version = that.getTemplateVersion(templateName);
-                //flavor = that.checkTemplateFlavor(templateName);
-                //description = that.getTemplateDescription(templateName);
+                // version = that.getTemplateVersion(templateName);
```

Comment:
Generally we do not keep commented code in source control. If there were a specific reason for this, it should be clearly communicated (with comment). I think we should clean this up.